src/modules/Security.rb

```diff
@@ -731,35 +733,30 @@ def Import(settings)
     # (For use by autoinstallation.)
     # @return [Hash] Dumped settings (later acceptable by Import ())
     def Export
-      Builtins.eval(@Settings)
+      @Settings if @Settings.is_a?(Hash)
     end
 
     # Create a textual summary and a list of unconfigured cards
     # @return summary of the current configuration
     def Summary
       settings = deep_copy(@Settings)
-      Builtins.foreach(@do_not_test) do |key|
-        settings = Builtins.remove(settings, key)
-      end
+      @do_not_test.each { |k| settings.delete k }
 
       # Determine current settings
       current = :custom
-      Builtins.maplist(@Levels) do |key, level|
-        Builtins.y2debug("%1=%2", key, level)
+      @Levels.each do |k, level|
+        Builtins.y2debug("%1=%2", k, level)
         current = key if level == settings
       end
+
       Builtins.y2debug("%1=%2", current, @Settings)
 
       # Summary text
       summary = _("Current Security Level: Custom settings")
       if current != :custom
         # Summary text
-        summary = Builtins.sformat(
-          _("Current Security Level: %1"),
-          Ops.get(@LevelsNames, Convert.to_string(current), "")
-        )
+        summary = _("Current Security Level: #{@LevelsNames[current]}")
```

Comment:
no, this won't work, as `_` method get extrapolated string, which is wrong. It have to look in ruby like
`summary = _("Current Security Level: %s") % @LevelsNames[current]`